.golangci.yml

```diff
@@ -29,136 +42,157 @@ linters:
+        # The next two are used in HTTP handlers, any error is handled by the server itself.
+        - io.WriteString
+        - (net/http.ResponseWriter).Write
+        # No need to check for errors on server's shutdown.
+        - (*net/http.Server).Shutdown
+        # Never check for rollback errors as Rollback() is called when a previous error was detected.
+        - (github.com/prometheus/prometheus/storage.Appender).Rollback
+    perfsprint:
+      # Optimizes `fmt.Errorf`.
+      errorf: true
+    revive:
+      # By default, revive will enable only the linting rules that are named in the configuration file.
+      # So, it's needed to explicitly enable all required rules here.
+      rules:
+        # https://github.com/mgechev/revive/blob/master/RULES_DESCRIPTIONS.md
+        - name: blank-imports
+        - name: comment-spacings
+        - name: context-as-argument
+          arguments:
+            # Allow functions with test or bench signatures.
+            - allowTypesBefore: '*testing.T,testing.TB'
+        - name: context-keys-type
+        - name: dot-imports
+        - name: early-return
+          arguments:
+            - "preserveScope"
+        # A lot of false positives: incorrectly identifies channel draining as "empty code block".
+        # See https://github.com/mgechev/revive/issues/386
+        - name: empty-block
+          disabled: true
+        - name: error-naming
+        - name: error-return
+        - name: error-strings
+        - name: errorf
+        - name: exported
+        - name: increment-decrement
+        - name: indent-error-flow
+          arguments:
+            - "preserveScope"
+        - name: package-comments
+          # TODO(beorn7): Currently, we have a lot of missing package doc comments. Maybe we should have them.
+          disabled: true
+        - name: range
+        - name: receiver-naming
+        - name: redefines-builtin-id
+        - name: superfluous-else
+          arguments:
+            - "preserveScope"
+        - name: time-naming
+        - name: unexported-return
+        - name: unreachable-code
+        - name: unused-parameter
+        - name: var-declaration
+        - name: var-naming
+    staticcheck:
+      checks:
+        - all # Enable all checks.
+        # FIXME: We should enable this check once we have fixed all the issues.
+        - -QF1001
+        - -QF1002
+        - -QF1003
+        - -QF1006
+        - -QF1007
+        - -QF1008
+        - -QF1009
+        - -QF1010
+        - -QF1012
+        - -ST1000
+        - -ST1003
+        - -ST1005
+        - -ST1012
+        - -ST1016
+        - -ST1020
+    testifylint:
+      disable:
+        - empty # FIXME
+        - equal-values # FIXME
+        - float-compare
+        - formatter # FIXME
+        - go-require
+        - len # FIXME
+        - useless-assert # FIXME: wait for golangci-lint > v2.0.2
+      enable-all: true
+
+output:
+  show-stats: false
+  sort-order:
+    - linter
+    - file
```

Comment:
I don't see the need to add `sort-order` configuration, since this is the default.
```suggestion
```